en/docs/developer-portal/manage-application.md

```diff
@@ -8,46 +8,150 @@ Let's get started!
+    - **Application access token expiry time:** The access token expiry time (seconds).
+    - **Refresh token expiry time:** The refresh token expiry time (seconds).
+    - **ID token expiry time:** ID token expiry time (seconds).
 
 5. Click **Generate Credentials** to generate the credentials for the application for the first time. 
 6. Once you generate the keys, you can find the application's consumer key and consumer secret. 
 
     ![Credentials](../assets/img/developer-portal/manage-applications/credentials.png){.cInlineImage-half}
 
 7. You can use the UI to generate a test token using the consumer key and consumer secret for the application, only for test purposes. We strongly recommend **NOT** using this token in your production environment. 
-8. Click **:// CURL** to copy the cURL command template for generating the access token. You can add the consumer key and consumer secret to the URL and invoke the token endpoint to generate an API access token.
+8. Click **:// CURL** to copy the cURL command template for generating the access token. You can generate an API access token by adding the consumer key and consumer secret to the URL and invoking the token endpoint.
+
+### Renew an Access Token using the Refresh Token Grant Type
+
+!!! note
+    - To use this grant type, you will need the refresh token you recieved when you generated your current access token.
+    - Your application must have the **Refresh Token** grant type enabled.
+    - You can use a refresh token only once.
+
+You can use a refresh token to obtain a new access token when your current access token expires or when you need to renew the access token due to security reasons. You need to use the [Refresh Token grant type](#refresh-token-grant) for this purpose. 
+
+To generate a new access token and a new refresh token, you can invoke the Token API by replacing the `<refresh-token>` with the refresh token you received with the access token and replacing `<base64(consumerKey:consumerSecret)>` with the application's **Consumer Key** value and **Consumer Secret** value as follows:
+
+``` java tab="Format"
+curl -k -d "grant_type=refresh_token&refresh_token=<refresh-token>" -H "Authorization: Basic <base64(consumerKey:consumerSecret)>" -H "Content-Type: application/x-www-form-urlencoded" https://sts.choreo.dev/oauth2/token
+```
+
+``` java tab="Example"
+curl -k -d "grant_type=refresh_token&refresh_token=3154090c-37f1-3268-90f9-8bd84daf135c" -H "Authorization: Basic UXk3RUZfVEtMbEVLWTlVRFpiWHVscVA4ZVVBYTpKSWN3VTlIX1hGUFdTcW1RQmllZ3lJUzRKazhh" -H "Content-Type: application/x-www-form-urlencoded" https://sts.choreo.dev/oauth2/token
+```
+
+``` java tab="Response"
+{
+    "scope":"default",
+    "token_type":"Bearer",
+    "expires_in":3600,
+    "refresh_token":"7ed6bae2b1d76c041787e8c8e2d6cbf8",
+    "access_token":"b7882d23f1f8207f4bc6cf4a20633ab1"
+}
+```
```

Comment:
Same as above, where you able to confirm these results?

Reply:
Yes I was.